pymc3/model_graph.py

```diff
@@ -35,10 +34,10 @@ def _get_ancestors(self, var, func) -> MutableSet[RV]:
         """
 
         # this contains all of the variables in the model EXCEPT var...
-        vars = set(self.var_list)
+        vars: MutableSet[RV] = set(self.var_list)
```

Comment:
Via #3466 i think we're staying away from variable annotations for now.

Reply:
@ColCarroll Oh, OK. Sorry to hear that. I use them extensively, and find them quite helpful.  May we use them in the comment form?

Reply:
Yes, their fine in comment form. But variable annotations are not supported by python 3.5 and we want to keep a tight integration with Arviz, which has to support python 3.5.